pkg/services/ppmshipment/ppm_shipment_fetcher.go

```diff
@@ -96,47 +96,47 @@ func FindPPMShipment(appCtx appcontext.AppContext, id uuid.UUID) (*models.PPMShi
-				"EmptyDocument.UserUploads.Upload",
-				"FullDocument.UserUploads.Upload",
-				"ProofOfTrailerOwnershipDocument.UserUploads.Upload")
-
-			if err != nil {
-				return apperror.NewQueryError("WeightTicket", err, "failed to load WeightTicket document uploads")
-			}
-
-			weightTicket.EmptyDocument.UserUploads = weightTicket.EmptyDocument.UserUploads.FilterDeleted()
-			weightTicket.FullDocument.UserUploads = weightTicket.FullDocument.UserUploads.FilterDeleted()
-			weightTicket.ProofOfTrailerOwnershipDocument.UserUploads = weightTicket.ProofOfTrailerOwnershipDocument.UserUploads.FilterDeleted()
+		weightTicket := &ppmShipment.WeightTickets[i]
+		err := appCtx.DB().Load(weightTicket,
+			"EmptyDocument.UserUploads.Upload",
+			"FullDocument.UserUploads.Upload",
+			"ProofOfTrailerOwnershipDocument.UserUploads.Upload")
+
+		if err != nil {
+			return apperror.NewQueryError("WeightTicket", err, "failed to load WeightTicket document uploads")
 		}
+
+		weightTicket.EmptyDocument.UserUploads = weightTicket.EmptyDocument.UserUploads.FilterDeleted()
+		weightTicket.FullDocument.UserUploads = weightTicket.FullDocument.UserUploads.FilterDeleted()
+		weightTicket.ProofOfTrailerOwnershipDocument.UserUploads = weightTicket.ProofOfTrailerOwnershipDocument.UserUploads.FilterDeleted()
 	}
 
+	ppmShipment.ProgearExpenses = ppmShipment.ProgearExpenses.FilterDeleted()
 	for i := range ppmShipment.ProgearExpenses {
-		if progearExpense := &ppmShipment.ProgearExpenses[i]; progearExpense.DeletedAt == nil {
-			err := appCtx.DB().Load(progearExpense,
-				"Document.UserUploads.Upload")
-
-			if err != nil {
-				return apperror.NewQueryError("ProgearExpenses", err, "failed to load ProgearExpenses document uploads")
-			}
+		progearExpense := &ppmShipment.ProgearExpenses[i]
+		err := appCtx.DB().Load(progearExpense,
+			"Document.UserUploads.Upload")
 
-			progearExpense.Document.UserUploads = progearExpense.Document.UserUploads.FilterDeleted()
+		if err != nil {
+			return apperror.NewQueryError("ProgearExpenses", err, "failed to load ProgearExpenses document uploads")
 		}
+
+		progearExpense.Document.UserUploads = progearExpense.Document.UserUploads.FilterDeleted()
 	}
 
+	ppmShipment.MovingExpenses = ppmShipment.MovingExpenses.FilterDeleted()
 	for i := range ppmShipment.MovingExpenses {
-		if movingExpense := &ppmShipment.MovingExpenses[i]; movingExpense.DeletedAt == nil {
-			err := appCtx.DB().Load(movingExpense,
-				"Document.UserUploads.Upload")
-
-			if err != nil {
-				return apperror.NewQueryError("MovingExpenses", err, "failed to load ProgearExpenses document uploads")
-			}
+		movingExpense := &ppmShipment.MovingExpenses[i]
+		err := appCtx.DB().Load(movingExpense,
+			"Document.UserUploads.Upload")
 
-			movingExpense.Document.UserUploads = movingExpense.Document.UserUploads.FilterDeleted()
+		if err != nil {
+			return apperror.NewQueryError("MovingExpenses", err, "failed to load ProgearExpenses document uploads")
```

Comment:
Little typo here (this was present before this PR, but might be worth fixing)
```suggestion
			return apperror.NewQueryError("MovingExpenses", err, "failed to load MovingExpenses document uploads")
```